internal/utils/docker.go

```diff
@@ -52,7 +51,18 @@ func AssertDockerIsRunning(ctx context.Context) error {
 }
 
 func DockerNetworkCreateIfNotExists(ctx context.Context, networkId string) error {
-	_, err := Docker.NetworkCreate(
+	existing, err := Docker.NetworkInspect(
+		ctx,
+		networkId,
+		types.NetworkInspectOptions{},
+	)
+
+	// if network already exists, abort
+	if existing.ID != "" && err == nil {
+		return nil
+	}
+
```

Comment:
Please also distinguish network not exist error from other errors like daemon unreachable. We want to proceed with creating the network only in former case and bubble up other errors.